litmus-portal/frontend/src/components/SideBar/styles.ts

```diff
@@ -22,7 +22,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     backgroundColor: `${theme.palette.cards.highlight} !important`,
     color: theme.palette.highlight,
     '& path': {
-      fill: theme.palette.highlight,
+      stroke: theme.palette.highlight,
```

Comment:
This is as per the design highlighting the sidebar Icon...any reason to change this one?

Reply:
This change -> ![Screenshot from 2021-06-22 11-10-34](https://user-images.githubusercontent.com/40641427/122869924-95d90300-d34a-11eb-80f4-1397f23a489d.png)

Figma design (present implementation) -> 
![Screenshot from 2021-06-22 11-05-45](https://user-images.githubusercontent.com/40641427/122870022-c325b100-d34a-11eb-84ac-bd1274a12f4d.png)

Reply:
The usage statistics svc icon is getting darkened when active if we use fill colour

Reply:
Can you please confirm that all others are not getting changed as the above one maybe? :sweat_smile: 

Else we might ask the deisgn team for a different svg later :thinking:

Reply:
removing my request for changes, just wanted to point that this might happen in case it got overlooked :+1:

Reply:
Only this one svg is looking a bit dark with the stroke change, others are fine.